### DISCUSSION TITLE
Release 1.2.0 Beta 3

### DISCUSSION THREAD

**Discussion by apohl79** [apohl79/audiogridder#537] [General]
## GENERAL

- Improved automation support
  - Bidirectional parameter syncing between AG plugin and DAW as well as AG plugin and AG server.
  - Support for recording automation runs via plugin UI as well as the generic editor.
  - Enables rendering of parameter movements in the generic editor and the remote UI.
  - This should resolve #485.

## PLUGIN

- Added "Keep Plugin UI Open" option
  - Allow the user to choose if the plugin UI should be opened automatically.
- Fixed settings menu #511.
- Minor UI fixes

## SERVER

- Hide cached plugins with a disabled format
- Fixed default I/O layout for plugin chains

<hr /><em>This discussion was created from the release <a href='https://github.com/apohl79/audiogridder/releases/tag/release_1_2_beta3'>Release 1.2.0 Beta 3</a>.</em>

**Comment by MrPisquan** [comment]
Hi,
first of all, thank you for your incredible work. This literally changed my life.
I just installed the beta 3 but the plugin is not connecting to the server.
[AudioGridderServer_2021-05-05_11-49-11.log](https://github.com/apohl79/audiogridder/files/6426681/AudioGridderServer_2021-05-05_11-49-11.log)
Thank you for being there!

**Comment by MrPisquan** [comment]
![IMG_7130](https://user-images.githubusercontent.com/75130976/117134190-2e1c2800-ad9d-11eb-9090-9b3dc890ccf9.jpg)

**Reply by apohl79** [reply]
Hm looks like a connection problem. It should not say "not connected". Can you check or share the logfile of the plugin?

**Reply by MrPisquan** [reply]
Hi Andreas,
yes I shared it in the previous message :-)
All the best

**Reply by apohl79** [reply]
That was the server log. The plugin log might show more insights.

**Reply by MrPisquan** [reply]
I restarted the computer and after the restart everything seems fine! Sorry Andreas, I think it was that session, I also installed/uninstalled other programs so my apologies, I should have thought to restart before writing!
There is always that question of the two servers instead of one, and I can't manage to automate parameters within Cubase 11.0.2, win10, but that annoying problem is sorted out.
Thank you again for the incredible support: a bunch of minutes and you were here to help!
Thank you
Federico

**Reply by apohl79** [reply]
Ok cool, you might have had loaded the prev version of the plugin. I should probably make the user feedback on connection problems a bit more readable (heard that before.. ;-)). 

> There is always that question of the two servers instead of one, and I can't manage to automate parameters within Cubase 11.0.2, win10, but that annoying problem is sorted out.

So are you saying automation is still broken for you? Or do I misread this?

> Thank you again for the incredible support: a bunch of minutes and you were here to help!

sure, after pushing a release I keep an eye open for immediate problems :-)

**Reply by MrPisquan** [reply]
Hi Andreas, thank you!
Yes I can't make the automation work. I put the track on rec mode (R/W enabled as well), started recording and moved some parameter from the plugin interface within Audio Gridder but nothing happens. I have tried several plugins.

**Reply by apohl79** [reply]
Ok i will have a look at cubase. I guess you have the parameters assigned to AG automation slots, correct?

**Reply by MrPisquan** [reply]
Sorry Andreas, what are the automation slots you are talking about? Surely that is the problem. I haven't assigned any slot, and I still don't know about their existence! :-))

**Reply by MrPisquan** [reply]
I've just loaded the plugin and changed some parameter from the UI within AG.

**Reply by apohl79** [reply]
check this: https://audiogridder.com/plugin/#automation

**Reply by MrPisquan** [reply]
Oh! There is nothing like that on Cubase... Unless I miss something!
![Capture](https://user-images.githubusercontent.com/75130976/117197150-5d538900-addf-11eb-8829-52e89a759fa4.PNG)

**Reply by apohl79** [reply]
This is independent of the DAW. You have to right-click on the insert within the AG plugin to open the context menu that has the Automation submenu.

**Reply by MrPisquan** [reply]
Found it! But still no automation is written even with the slot enabled.

**Reply by apohl79** [reply]
hmm working normally in cubase 10. here is how it is supposed to work:

1) assign the param you want to automate to a slot in AG.
2) hit the automation "W" button of the channel
3) play and change the parameter - alternatively, edit the automation curve manually.
4) hit the automation "R" button of the channel

**Reply by MrPisquan** [reply]
Hi Andreas,
mmmh. This time I did everything right.
1) the preset list is showing but when I select one preset nothing happens (several presets and plugin tested)
2) I assigned the slot, hit the Write automation button, changed the parameter but the DAW detects no automation.
3) I tried then to automate it from the daw, but the automation doesn't affect the plugin.
Thank you for your support!
Federico

**Reply by apohl79** [reply]
Would you be able to record a little screen video showing how you do this?

**Reply by MrPisquan** [reply]
Yes with pleasure, here it is:

https://wetransfer.com/downloads/1bd550127264aed8f5c0c9c39cffc80920210506194756/8bb9a3f090511cb191942c24d6bae91920210506194756/aa10c7

Thank you so much!

Federico

**Reply by MrPisquan** [reply]
Today I've dug more in the matter. Depending on the plugin the behaviour of AG is different, You will see in the video above some examples.
:-)
F.

**Comment by fojerhar** [comment]
Woow

A damn happy user here. Now working flawlessy in my hack with protools 21.3.1. Everything. Open close, as many instances I want (I could use only one in a session), plugin copy also works. old session opens up.
So, thank you for the amazing job, changes life, no stem mixdowns, no cpu peaks, all 32 cores have things to do :)))
Even the acustica modula works trough this troug two juce engines, funny
Big claps (my GF also happy, because I'm happy)
Looking again for the ppal button.
Thankyou
Viktor

**Reply by apohl79** [reply]
> my GF also happy, because I'm happy

:-D

**Comment by CaBuffalo** [comment]
Hi, Victor and Federico,

Thanks for the feedback. We are glad to see new Beta 3 solves the problem.

Enjoy your mixing and please help us by spreading the words.

Cheers.

On Wed, May 5, 2021 at 7:39 AM Andreas Pohl ***@***.***>
wrote:

> Ok cool, you might have had loaded the prev version of the plugin. I
> should probably make the user feedback on connection problems a bit more
> readable (heard that before.. ;-)).
>
> There is always that question of the two servers instead of one, and I
> can't manage to automate parameters within Cubase 11.0.2, win10, but that
> annoying problem is sorted out.
>
> So are you saying automation is still broken for you? Or do I misread this?
>
> Thank you again for the incredible support: a bunch of minutes and you
> were here to help!
>
> sure, after pushing a release I keep an eye open for immediate problems :-)
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/537#discussioncomment-699087>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6EEFUTVBSBF5T4XK7TTMFKDBANCNFSM44ELW3TA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by AaronNiduk** [comment]
Hello) Sorry for my bad english
You saved me and my Xeon 2689 by using AudioGridder as a bridge for my plugins.
I am very grateful to you and will soon receive a scholarship and donate to you!))

I will have a request (I think many have it too)
Unfortunately, when you switch to the daw, the plugin is minimized.
Would like a * detached * function for the plugin
Please add the dock function of the plugin window.
I sent you a video.
Thanks!)))

https://user-images.githubusercontent.com/83723053/117183174-e66ac000-addf-11eb-9e18-6b0db3b3d76d.MP4

**Reply by apohl79** [reply]
I see, try moving the ag plugin DAW window (assuming local mode enabled). This should bring the plugin UI back to the front. 

The way AG works is a bit limiting for running the server and DAW on the same desktop, as the plugin window belongs to the server process while the ag plugin window belongs to the DAW process.

One thing I will try though is forcing and keeping the plugin windows owned by the server to the front of the desktop. Right now this happens only when the position changes.

**Comment by fojerhar** [comment]
Hi amazing team

An issue with the 1.2.0.b3
Computer: AMD hack with catalina, pro tools 21.3.1

At sample rates 44.1 and 48K everything works like a dream, fast, reliable, and AG takes out all the processing power from PT
<img width="466" alt="44 1K" src="https://user-images.githubusercontent.com/72139943/117421250-a65e2700-af1e-11eb-89f7-36cdece2aad9.png">
The same converted session to 96K with the same plugins do not takes out the cpu needs from PT
<img width="475" alt="96K" src="https://user-images.githubusercontent.com/72139943/117421373-c68de600-af1e-11eb-82b2-1b05d10dea41.png">
And I have cpu peaks and playback stops also
Here are the traces logs
[AudioGridder 44-96K.zip](https://github.com/apohl79/audiogridder/files/6440156/AudioGridder.44-96K.zip)
First I opened the 96K session, then the 44.1K session

Thank you
Viktor

**Comment by jin2** [comment]
I like the direction you guys are taking. Bravo for the continuous development. 

However I'm getting a lot of crashing with both beta 2 and 3. I'm loading old projects that used 1.1 and after installs of the betas i eventually get into a cycle of crashing (the server goes into continual reboot) 

going back to 1.1 resolves this. 

Is there anything I can do to prevent this?

**Reply by apohl79** [reply]
Please provide bug report(s) for crashes. https://audiogridder.com/bug-reports/

Thanks!

**Reply by jin2** [reply]
[AG Log.zip](https://github.com/apohl79/audiogridder/files/6487837/AG.Log.zip)

Hi,
So installed beta 3 yesterday and used it for a few hours with no issues at all.

I open up the same session in Logic x today and I get a crash as soon as I press play.

I then emptied the AG log folders and re-ran the scenario above, the attached logs are from the most recent incident.

What was different this time was having to press allow incoming connections for AG, i dont recall having to do that at installation.

Hope it helps.


Thanks

**Comment by ignaceland** [comment]
Hello AudioGridder team, thanks for making this very useful and well developed software. 

Just wanted to report that I am having the following problem in 1.2.0 beta3 with every plugin in Harrison Mixbus. 
It doesn't happen with version 1.1.1. I think it has to do with the multichannel support on the new version, though as far as I can see everything looks fine on the pin connections. 

![image](https://user-images.githubusercontent.com/44209622/117583245-84120800-b0dc-11eb-8ad8-d3264743be92.png)
![image](https://user-images.githubusercontent.com/44209622/117583253-90966080-b0dc-11eb-8138-1b4fc9d2640e.png)

The 1.2.0 beta3 loads the same plugins fine here in REAPER 6.
Kind regards

**Reply by apohl79** [reply]
Could you provide the plugin/server logs for the failed runs? https://audiogridder.com/bug-report-diagnostics-locations/

**Reply by ignaceland** [reply]
Sure thing
[AudioGridderServer_2021-05-10_12-42-59.log](https://github.com/apohl79/audiogridder/files/6453484/AudioGridderServer_2021-05-10_12-42-59.log)
[AudioGridderPlugin_2021-05-10_12-43-47.log](https://github.com/apohl79/audiogridder/files/6453488/AudioGridderPlugin_2021-05-10_12-43-47.log)

**Comment by Dirxman** [comment]
Hello,

maybe related to other I/O related problems....
The VST3 version of Audiogridder 1.2.0 beta3 comes up in an Amisonics 3rd Order Channel in Cubase Pro11, and even in Cubase Elements11 (although it a non valid Channel config in Elements, shouldn't even be possible there).

Cheers,
Dirk

**Reply by apohl79** [reply]
An amisonics 3rd order cubase channel uses 16 (mono) channels. As AG can do 16 in/out, why should it not be there?

**Reply by Dirxman** [reply]
Hi,

I forgot to mention that I use the Instrument Plugin.
I presume there is something wrong, as I did not want to use 3rd Order Ambisonics.
It opens as 3rd Order Ambisonics in an Application that is not supposed to allow this (Cubase Elements). 

If I open the Instrument, Cubase (Pro and Elements) asks for output Channel config, it is set to stereo.
In both apps the result is an Instrument Track with 3rd Order Ambisonics configuration.
No Server involved at this point.

Cheers,
Dirk

**Reply by apohl79** [reply]
understood, ok thx, I will have a look at this

**Comment by djschultz123** [comment]
I'm stuck using 1.1 because I can't get any audio output from AG with the 1.2 betas in Studio One. Not sure what changed (nothing did on my end). The 1,2 beta FX plugin works great. But the instrument one, not so much. I can see that the synths are triggered, but no sound is produced out the other side.

Win10 latest release build
Studio One latest release build

**Comment by fojerhar** [comment]
Hi
Have found a bug in your really nice and amazing 1.2.0.b3 release.
Catalina, Protools
If I automate a master bypass for AG plugin, and I stop the playback where the automation is off then server and plugin collapses. If I let run trough the on off section it is fine, only if I stop the playback there
[AudioGridder.zip](https://github.com/apohl79/audiogridder/files/6524624/AudioGridder.zip)
thank you for the amazing work!

**Comment by bluesbop** [comment]
Thank you guys for all your work on this, it's made an unbelievable difference in my workflow.

I'm finding that AU plugs are much less stable in beta3 compared to 1.1.1. I can't even open a single Acustica plug without the server going into a crash loop, but can load a bunch of VST versions without problem, even the ZL ones. Fabfilter AUs would crash the server twice and then settle down and work fine, and I also couldn't open Gullfoss AU...but VST versions of those plugs did not crash the server. 

Anyone else noticing this?
